### PR TITLE
Add tests for cart

### PR DESCRIPTION
**Pull request by noahlissner**

### CODE REVIEW

File: cypress/integration/test.spec.ts [hunk 1]
```diff
@@ -3,3 +3,32 @@ describe("My First Test", () => {
     expect(true).to.equal(true);
   });
 });
+
+describe("Cart - Tests", () => {
+  it("Open/Close Cart", () => {
+    cy.visit('http://localhost:1234')
+
+    cy.get('.cart__btn').click()
```
Comment (dennislundgren) [inline cypress/integration/test.spec.ts:11]: See row 18, 28 - different class names.

File: cypress/integration/test.spec.ts [hunk 2]
```diff
@@ -3,3 +3,32 @@ describe("My First Test", () => {
...
+
+    cy.get('.cart').should('have.class', 'open')
+  })
+  it("Increase cart quantity", () => {
+    cy.visit('http://localhost:1234')
+
+    cy.get('.cart-__btn').click()
```
Comment (dennislundgren) [inline cypress/integration/test.spec.ts:18]: Change class name.

File: cypress/integration/test.spec.ts [hunk 3]
```diff
@@ -3,3 +3,32 @@ describe("My First Test", () => {
...
+
+    cy.get('.increase').click()
+
+    cy.get('.quantity').should('equal', '2')
+  })
+
+  it("Decrease cart quantity", () => {
+    cy.visit('http://localhost:1234')
+
+    cy.get('.cart-btn').click()
```
Comment (dennislundgren) [inline cypress/integration/test.spec.ts:28]: Change class name.

File: cypress/integration/test.spec.ts (hunk 1 above)
Comment (dennislundgren) [reply cypress/integration/test.spec.ts:11]: Use: .cart__btn

**Review by dennislundgren — COMMENTED** [review]
Please check comments.

**Review by dennislundgren — APPROVED** [review]
Very good. I approve.

**Review by elilyx10 — COMMENTED** [review]
Good job!

**Pull request merged**